ccmlib/scylla_docker_cluster.py

```diff
@@ -0,0 +1,53 @@
+import os
+from subprocess import run
+
+from ccmlib.scylla_cluster import ScyllaCluster
+from ccmlib.scylla_node import ScyllaNode
+
+
+class ScyllaDockerCluster(ScyllaCluster):
+    def __init__(self, *args, **kwargs):
+        super(ScyllaDockerCluster, self).__init__(*args, **kwargs)
+        self.docker_image = kwargs['docker_image']
+
+    def create_node(self, name, auto_bootstrap, thrift_interface,
+                    storage_interface, jmx_port, remote_debug_port,
+                    initial_token, save=True, binary_interface=None):
+
+        return ScyllaDockerNode(name, self, auto_bootstrap, thrift_interface,
+                                storage_interface, jmx_port, remote_debug_port,
+                                initial_token, save=save, binary_interface=binary_interface,
+                                scylla_manager=self._scylla_manager)
+
+
+class ScyllaDockerNode(ScyllaNode):
```

Comment:
shouldn't this class go under `ccmlib/scylla_docker_node.py` instead of here?

Reply:
Probably, I'm just trying to sketch things up first, splitting to a new file would take 2 min...